README.md

```diff
@@ -1,295 +1,202 @@
-```
-
-Now try:
+Try copying the file we just created:
 
 ```bash
-$ ls
+$ cp hello_world.rb hello_world_again.rb
 ```
 
-You should see the file you just created, `hello_world.rb`, in the working directory.
-Note that this is an empty file and has nothing inside of it, because you just created it.
+* If the command contains *two* file names, then it copy the contents of 1st
+  file to the 2nd file. If the 2nd file doesn’t exist, then first it creates one
+  and content is copied to it. But if it existed then it is simply overwritten
+  without any warning. So be careful when you choose destination file name.
 
-### Making New Directiries Using `mkdir`
-
-We can make directories with the `mkdir` command:
+* If the command has one or more arguments, specifying file names and following
+  those arguments, an argument specifying directory name then this command
+  copies each source file to the destination directory with the same name. It
+  will create the directory if it doesn't exist, but **if it exists then it will
+  be overwritten**, so be careful!
 
 ```bash
-$ mkdir name_of_directory
+$ cp hello_world.rb hello_world_again.rb name_of_directory
 ```
-Now if you enter `ls` you should see the empty directory you just created in the working
-directory.
+Now you have too many files, or some of them are no longer needed. Let's talk
+about how to remove files.
 
 ### Removing Files With `rm`
 
-To delete a file, we can enter `rm` at a shell prompt.
-**Note:** Deleting a file with rm is *permanent*. This action cannot be undone.
+To delete a file, we can enter `rm` at a shell prompt. This command stands for
+**r**e**m**ove.
 
 ```bash
 $ rm hello_world.rb
 ```
+**Note:** Deleting a file with rm is *permanent*. This action cannot be undone.
 
 There additional options to `rm`:
 
-* -i (interactive) — Prompts you to confirm the deletion. This option can stop you from
-deleting a file by mistake.
-* -f (force) — Overrides interactive mode and removes the file(s) without prompting.
-Use this with caution. This action cannot be undone!
+* -i (interactive) — Prompts you to confirm the deletion. This option can stop
+  you from deleting a file by mistake.
+* -f (force) — Overrides interactive mode and removes the file(s) without
+  prompting. Use this with caution. This action cannot be undone!
 * -v (verbose) — Shows the progress of the files as they are being removed.
-* -r (recursive) — Deletes a directory and all files and subdirectories it contains.
+* -r (recursive) — Deletes a directory and all files and subdirectories it
+  contains.
+* Combining the flags `-rf` will do a recursive force delete.
+
+You can also delete directories using `rm` or `rmdir`. If you want to remove all
+folders and files in a directory the dull command is `rm -rf name_of_directory`.
 
-You can also delete directories using `rm` or `rmdir`.
+**Note:** When using `rm -rf`, be _very_ careful. If you submit a command like
+`rm -rf /` , if you will delete the root directory!
```

Comment:
you will delete, recursively, everything from the root directory downward – effectively wiping your hard drive. This would be bad. There are some builtins to limit the damage of this action, but under the right conditions, you could effectively scorch your hard disk. This command, in particular, is often a "joke" amongst programmer because it is so very, very, very bad to do.